pkg/testhelper/testhelper.go

```diff
@@ -27,6 +31,90 @@ const (
 	ERROR
 )
 
+type ObjectOut struct {
+	ObjectType   string
+	ObjectFields map[string]string
+}
+
+type FailureReasonOut struct {
+	CompliantObjectsOut    []*ObjectOut
+	NonCompliantObjectsOut []*ObjectOut
+}
+
+const (
+	Namespace              = "Namespace"
+	PodName                = "PodName"
+	ContainerName          = "ContainerName"
+	ProcessID              = "ProcessID"
+	ProcessCommandLine     = "ProcessCommandLine"
+	SchedulingPolicy       = "SchedulingPolicy"
+	SchedulingPriority     = "SchedulingPriority"
+	ReasonForNonCompliance = "ReasonForNonCompliance"
+	ReasonForCompliance    = "ReasonForCompliance"
+	Category               = "Category"
+)
+
+const (
+	UndefinedType        = "Undefined Type"
+	CnfType              = "Cnf"
+	PodType              = "Pod"
+	ContainerType        = "Container"
+	ContainerProcessType = "ContainerProcess"
+)
+
+func NewProcessObjectOut(aContainer *ObjectOut, aPolicy, aPriority, aCommandLine string) (out *ObjectOut) {
```

Comment:
I left it out on purpose. In my opinion, the PID is not very helpful in the report as it is dynamic. The information identifying better the process is the command line. A Partner looking at it will know exactly which pod/container is identified.